src/Runner.jl

```diff
@@ -925,17 +927,21 @@ function platform_envs(platform::AbstractPlatform, src_name::AbstractString;
     # so we set all the environment variables that we've seen them called
     # and hope for the best.
     for host_map in (tool -> "HOST$(tool)", tool -> "$(tool)_FOR_BUILD", tool -> "BUILD_$(tool)", tool -> "$(tool)_BUILD")
+        # Use full path to avoid collisions when the target is similar to the
+        # host (e.g., `x86_64-linux-musl-cxx03` and `x86_64-linux-musl-cxx11`)
+        host_bin_dir = "/opt/bin/$(aatriplet(host_platform))"
```

Comment:
I think you don't want to use `aatriplet()` but instead just `triplet()`.  Since `aatriplet()` should strip out the abi parts.  (`aa` means `"abi-agnostic"`)

Reply:
These are just environment variables, those paths come from the RootFS

Reply:
It turned out these paths didn't come from the RootFS, but the fact that they were implicitly ABI agnostic was because currently the host platform is always `x86_64-linux-musl`, without specification of the ABI.